From c98a4fd68d3e4a81cf6102b89029e90bbfc90437 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?= <avarab@gmail.com>
Date: Sun, 14 Nov 2010 15:56:39 +0100
Subject: [PATCH 002/160] Makefile: Set NO_GETTEXT=YesPlease on Windows & MinGW
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Change the Windows and MinGW build defaults to not build the gettext
tools by default. Gettext hasn't yet been ported to the default
environments on those systems, although Erik Faye-Lund is working on
MinGW support.

Reported-by: Johannes Sixt <j6t@kdbg.org>
Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 Makefile |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/Makefile b/Makefile
index 426e0b1..67d94d3 100644
--- a/Makefile
+++ b/Makefile
@@ -826,6 +826,8 @@ ifeq ($(uname_S),Windows)
 	NO_REGEX = YesPlease
 	NO_CURL = YesPlease
 	NO_PYTHON = YesPlease
+	NO_GETTEXT = YesPlease
+	NEEDS_LIBINTL =
 	BLK_SHA1 = YesPlease
 	NATIVE_CRLF = YesPlease
 
@@ -878,6 +880,8 @@ ifneq (,$(findstring MINGW,$(uname_S)))
 	OBJECT_CREATION_USES_RENAMES = UnfortunatelyNeedsTo
 	NO_REGEX = YesPlease
 	NO_PYTHON = YesPlease
+	NO_GETTEXT = YesPlease
+	NEEDS_LIBINTL =
 	BLK_SHA1 = YesPlease
 	ETAGS_TARGET = ETAGS
 	COMPAT_CFLAGS += -D__USE_MINGW_ACCESS -DNOGDI -Icompat -Icompat/fnmatch -Icompat/win32
-- 
1.7.2.3

